Tasks/VSMobileCenterUpload/task.json

```diff
@@ -68,7 +68,17 @@
             "groupName": "symbols",
             "required": false,
             "helpMarkDown": "Relative path from the repo root to the symbols folder.",
-            "visibleRule": "symbolsType = Windows || symbolsType == AndroidNative"
+            "visibleRule": "symbolsType == AndroidNative"
+        },
+        {
+            "name": "pdbPath",
+            "type": "filePath",
+            "label": "PDB Files Path Pattern",
```

Comment:
Suggestion for this label:
"Symbols Path (*.pdb)"
Then users have a hint, in addition to the helpMarkDown, about what PDB files are.